oauthenticator/tests/test_google.py

```diff
@@ -66,3 +66,40 @@ def google_client(client):
     with raises(HTTPError) as exc:
         name = await authenticator.authenticate(handler)
     assert exc.value.status_code == 403
+
+
+async def test_user_in_groups(google_client):
+    authenticator = GoogleOAuthenticator(
+        hosted_domain=['email.com', 'mycollege.edu'],
+        admin_google_groups={'email.com': ['fakeadmingroup']},
+        google_group_whitelist={'email.com': ['fakegroup']}
+    )
+    admin_handler = google_client.handler_for_user(user_model('fakeadmin@email.com'))
+    admin_user_info = await authenticator.authenticate(admin_handler, google_groups=['anotherone', 'fakeadmingroup'])
+    user_is_admin = admin_user_info['admin']
+    assert user_is_admin == True
+    whitelist_handler = google_client.handler_for_user(user_model('fakewhitelisted@email.com'))
+    whitelist_user_info = await authenticator.authenticate(whitelist_handler, google_groups=['anotherone', 'fakegroup'])
+    user_is_whitelisted = whitelist_user_info['auth_state']['google_user']['google_groups']
+    user_is_not_admin = whitelist_user_info['admin']
+    assert 'fakegroup' in user_is_whitelisted
+    assert user_is_not_admin == False
+    non_whitelist_handler = google_client.handler_for_user(user_model('fakenonwhitelisted@email.com'))
+    non_whitelist_user_info = await authenticator.authenticate(non_whitelist_handler, google_groups=['anotherone', 'fakenonwhitelistedgroup'])
+    assert non_whitelist_user_info is None
+    non_admin_authenticator = GoogleOAuthenticator(
```

Comment:
What do you think about separating this into two tests?
* `test_admin_google_groups`
* `test_whitelisted_google_groups`

Reply:
done!